### PR TITLE
Add Airflow DAGs

### PR DESCRIPTION
**Pull request by msumit**

### CODE REVIEW

File: airflow/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Demotrends using Apache Airflow
+
+If you want to use [Apache Airflow](https://github.com/apache/incubator-airflow) to manage the Demotredns workflow, you can make use of following 2 python scripts:
+
+## process_static_data.py
+Should run once to create `page` and `redirect` table in Hive. Ideally one should download [enwiki-latest-page.sql.gz](http://dumps.wikimedia.org/enwiki/latest/enwiki-latest-page.sql.gz) and [enwiki-latest-redirect.sql.gz](http://dumps.wikimedia.org/enwiki/latest/enwiki-latest-redirect.sql.gz) and load them in mysql instance and then use DB export (Sqoop) to populate the data for these 2 hive tables. But for simplicity Qubole has done it and placed data on S3. One just need to create hive tables against them.
```
Comment (yogesh2021) [inline airflow/README.md]: fix: DB Import

File: airflow/process_daily_data.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+from airflow.contrib.operators.qubole_operator import QuboleOperator
+from datetime import datetime, timedelta
+
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': True,
+    'start_date': datetime(2016, 8, 1),
+    'end_date': datetime(2016, 8, 5)
+    'email': ['sumitm@qubole.com'],
```
Comment (yogesh2021) [inline airflow/process_daily_data.py]: Use `airflow@qubole.com`

File: airflow/README.md (hunk 1 above)
Comment (msumit) [reply airflow/README.md]: fixed

File: airflow/process_daily_data.py (hunk 2 above)
Comment (msumit) [reply airflow/process_daily_data.py]: changed

**Comment by yogesh2021** [comment]
lgtm & awesome work, this is a proper ETL use case, let's merge it & show it as part of upcoming Examples Documentation

File: airflow/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Demotrends using Apache Airflow
+
+If you want to use [Apache Airflow](https://github.com/apache/incubator-airflow) to manage the Demotredns workflow, you can make use of following 2 python scripts:
```
Comment (vrajat) [inline airflow/README.md]: Typo: `Demotrends` instead of `Demotredns`

**Review by vrajat — CHANGES_REQUESTED** [review]
A minor typo in README needs to be fixed.

**Review by vrajat — APPROVED** [review]

**Pull request merged**